### PR TITLE
[ADD] Ported/Added partner_academic_title for Odoo 9 - did work out of the box

### PR DESCRIPTION
**Pull request by wpichler**

### CODE REVIEW

File: partner_academic_title/README.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+This module was written to adds possibility to define some academic titles and
+to add multiple titles on partners. there is a sequence on titles to display
+them in an order on the partner.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/8.0
```
Comment (nikul-serpentcs) [inline partner_academic_title/README.rst]: @wpichler IMO Change Target.

File: partner_academic_title/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#     along with partner_academic_title.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Partner Academic Title",
+    'summary': """
+        Add possibility to define some academic title""",
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```
Comment (nikul-serpentcs) [inline partner_academic_title/__openerp__.py]: @wpichler Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: partner_academic_title/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (nikul-serpentcs) [inline partner_academic_title/__openerp__.py]: @wpichler Change copyright like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L2)

EX.` # Copyright <YEAR(S)> <AUTHOR(S)>`

File: partner_academic_title/views/res_partner_view.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+            <field name="inherit_id" ref="base.view_partner_form" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='title']" position="after">
+                    <field name="academic_title_ids" class="oe_edit_only" widget="many2many_tags" placeholder="Academic Titles" attrs="{'invisible': [('is_company', '=', True)]}"/>
+                    <field name="academic_title_display" class="oe_read_only" attrs="{'invisible': [('is_company', '=', True)]}" />
+                </xpath>
+            </field>
+        </record>
+    </data>
+</openerp>
```
Comment (nikul-serpentcs) [inline partner_academic_title/views/res_partner_view.xml]: add new blank line

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: partner_academic_title/README.rst (hunk 1 above)
Comment (wpichler) [reply partner_academic_title/README.rst]: is runbot target needed ? i will remove it when not needed - no idea how to get the new runbot target

File: partner_academic_title/__openerp__.py (hunk 2 above)
Comment (wpichler) [reply partner_academic_title/__openerp__.py]: done

File: partner_academic_title/__openerp__.py (hunk 3 above)
Comment (wpichler) [reply partner_academic_title/__openerp__.py]: done

File: partner_academic_title/views/res_partner_view.xml (hunk 4 above)
Comment (wpichler) [reply partner_academic_title/views/res_partner_view.xml]: done

**Comment by wpichler** [comment]
requested changes done - please review

File: partner_academic_title/views/partner_academic_title_view.xml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        </record>
+        <record id="open_academic_title" model="ir.actions.act_window">
+            <field name="name">Academic Titles</field>
+            <field name="res_model">partner.academic.title</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+        </record>
+        <menuitem id="menu_academic_title" name="Academic Titles" parent="base.menu_config_address_book" groups="base.group_system" action="open_academic_title" sequence="4"/>
+    </data>
+</openerp>
```
Comment (nikul-serpentcs) [inline partner_academic_title/views/partner_academic_title_view.xml]: Add new blank line

**Review by nikul-serpentcs — APPROVED** [review]
small Change,Code Review LGTM

**Comment by wpichler** [comment]
now with blank new lines

**Pull request merged**